apps/site/pages/en/blog/community/2025-06-28-Emelia-Smith.md

```diff
@@ -0,0 +1,27 @@
+# Node.js LGBTQIA+ Stories: Emelia Smith
+
+_how did you come to understand who you are, and what contributions have you made to open source?_
```

Comment:
```suggestion
```

Reply:
That's not desired to be in the articles?